tsdb/mapper.go

```diff
@@ -306,7 +338,7 @@ func (lm *LocalMapper) nextChunkRaw() (*MapperOutput, error) {
 // for the current tagset. Tagsets are always processed in the same order as that
 // returned by AvailTagsSets(). When there is no more data for any tagset nil
 // is returned.
-func (lm *LocalMapper) nextChunkAgg() (*MapperOutput, error) {
+func (lm *LocalMapper) nextChunkAgg() (interface{}, error) {
```

Comment:
This one really took me by surprise.  I didn't know that both the type and value of an `interface{}` had to be `nil` for `if v == nil` to be `true`.